client/tests/integration/permissions.rs

```diff
@@ -19,6 +19,40 @@ fn get_assets(iroha_client: &mut Client, id: &AccountId) -> Vec<Asset> {
         .expect("Failed to execute request.")
 }
 
+#[test]
+fn permissions_require_registration_before_grant() {
+    let (_rt, _peer, iroha_client) = <PeerBuilder>::new()
+        .with_instruction_judge(public_blockchain::default_permissions())
```

Comment:
I don't think it is, `new()` just forwards to `default()` which has [None](https://github.com/hyperledger/iroha/blob/33ab59a1c6c3f4e85a842df3fbbdfbdcfe38fdf6/core/test_network/src/lib.rs#L728) for instruction_judge

Reply:
Hm... I was confused at a moment. Yes, builder has `None` by default, but then it creates `Peer` it will is `default_permissions()` if user hasn't provided any. Please, look at `start_with_peer()` method and where it is used

Reply:
I think that being explicit here makes it far more readable.